<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_partie3_2</class>
 <widget class="QDialog" name="fct_partie3_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des résultats aux épreuves</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>401</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="table_3_2">
      <column>
       <property name="text">
        <string>numEp</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>gold</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>silver</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>bronze</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Valider_ajouter">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_numEp">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_gold">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>105</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_3_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>695</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>75</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>numEp</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un résultat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>212</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Supprimer un résultat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Résultat sélectionné :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_numEp_affiche">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_gold_affiche">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider_supprimer">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modifier un résultat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modif_gold">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>552</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modif_numEp">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>516</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider_modifier">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_modif">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_supprimer">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur_ajouter">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>38</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Actualiser">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>gold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>145</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>silver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>175</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>bronze</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_silver">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_bronze">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>173</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>numEp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>gold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>silver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>bronze</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_silver_affiche">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_bronze_affiche">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>silver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>625</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>bronze</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>numEp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>555</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>gold</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modif_silver">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>586</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modif_bronze">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>table_3_2</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>fct_partie3_2</receiver>
   <slot>afficherResultat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Actualiser</sender>
   <signal>clicked()</signal>
   <receiver>fct_partie3_2</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valider_ajouter</sender>
   <signal>clicked()</signal>
   <receiver>fct_partie3_2</receiver>
   <slot>ajouterResultat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valider_supprimer</sender>
   <signal>clicked()</signal>
   <receiver>fct_partie3_2</receiver>
   <slot>supprimerResultat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Valider_modifier</sender>
   <signal>clicked()</signal>
   <receiver>fct_partie3_2</receiver>
   <slot>modifierResultat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>afficherInscription()</slot>
  <slot>modifierInscription()</slot>
  <slot>ajouterInscription()</slot>
  <slot>supprimerInscription()</slot>
  <slot>afficherResultat()</slot>
  <slot>ajouterResultat()</slot>
  <slot>supprimerResultat()</slot>
  <slot>modifierResultat()</slot>
 </slots>
</ui>
